forum/qa-include/qa-theme-base.php

```diff
@@ -1730,10 +1730,35 @@ public function vote_clear()
 
 	public function a_count($post)
 	{
-		// You can also use $post['answers_raw'] to get a raw integer count of answers
+		$comments_attr = null;
+		if (qa_request() === 'activity') {
+			$comments = 0;
+			$answers_comments = '';
+			$postid = $post['raw']['postid'];
+			$sql = 'SELECT postid, type FROM ^posts WHERE parentid=# AND (type="C" OR type="A")';
+			$question_posts = qa_db_read_all_assoc(qa_db_query_sub($sql, $postid));
+			foreach ($question_posts as $qpost) {
+				if (!empty($qpost['postid'])) {
+					if ($qpost['type'] === 'C') {
+						$comments++;
+					} else {
+						$answers_comments .= (!empty($answers_comments) ? ',' : '').$qpost['postid'];
+					}
+				}
+			}
+			if (!empty($answers_comments)) {
+				$sql = 'SELECT COUNT(postid) AS comments FROM ^posts WHERE parentid IN ('.$answers_comments.') AND type="C"';

```

Comment:
a może jedno zapytanie które liczy wszystko to co jest potrzebne? W końcu ruch nie jest mały, a cache jak nie było tak nie ma :)